FinancialTracker/Firebase/DatabaseManager.swift

```diff
@@ -304,13 +303,75 @@ class DatabaseManager {
                     completionHandler(DatabaseError.database(error), false)
                     return
                 }
-                completionHandler(nil, true)
+                
+                self.changeTransfersCurrency(currency: currency, uid: currentUser.uid) { databaseError, success in
+                    completionHandler(databaseError, success)
+                }
             }
         } catch {
             completionHandler(DatabaseError.database(error), false)
         }
     }
     
+    /// Changing current user's currency to a new one of his choice.
+    /// - Parameters:
+    ///   - currency: A `Currency` type containing the currency's symbol, rate, code and name.
+    ///   - uid: A `String` type containing the currentUser uid.
+    ///   - completionHandler: Block that is to be executed if an error appears or the function is successfully executed.
+    ///     1. `databaseError` - An error object that indicates why the function failed, or nil if the was successful.
+    ///     2. `success` - Boolean that indicates whether the function was successful.
+    private func changeTransfersCurrency(currency: Currency, uid: String, completionHandler: @escaping(DatabaseError?, Bool) -> Void) {
+        let usersKey = DBCollectionKey.users.rawValue
+        let transfersKey = DBCollectionKey.transfers.rawValue
+        
+        firestore.collection(usersKey).document(uid).collection(transfersKey).getDocuments { querySnapshot, error in
+            if let error = error {
+                completionHandler(DatabaseError.database(error), false)
+                return
+            }
+
+            guard let documents = querySnapshot?.documents else {
+                completionHandler(DatabaseError.unknown, false)
```

Comment:
There is no error type to be given to `DatabaseError.database()`

Reply:
When i think about it, it should be an empty return because when it doesn't query anything (e.g. there are no current transfers it shouldn't present an error. P.S. Now I understand why I got an unknown alert.)

Reply:
First of all, "Now I understand why I got an unknown alert" comment - if you do not use an unknown error, or put an unknown error with description, you will know right away what is going on, not after I PR review your changes. 

Look at the completionHandler's description - `documentSet` is actually `QuerySnapshot` and it says that it cannot be nil unless there is an error.

In other words, you may have `querySnapshot?.documents` count that is 0, but if `querySnapshot?` is actually `nil` then you must have an error of some kind. 

<img width="665" alt="Screenshot 2022-03-21 at 10 17 19" src="https://user-images.githubusercontent.com/18184558/159224946-c66a5045-ebe3-4538-9674-652ab5dc3e71.png">

Reply:
Because I am querying for collection which when I create an user doesn't exist.